docs/docusaurus/docs/cloud/connect/connect_postgresql.md

```diff
@@ -0,0 +1,179 @@
+---
+sidebar_label: 'Connect GX Cloud to PostgreSQL'
+title: 'Connect GX Cloud to PostgreSQL'
+description: Connect GX Cloud to a PostgreSQL Data Source.
+---
+
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
+
+To validate data stored in a PostgreSQL database from GX Cloud, you must add the GX Agent to your deployment environment. The GX Agent acts as an intermediary between GX Cloud and PostgreSQL and allows you to securely access and validate your data in GX Cloud.
+
+New to GX Cloud and not sure that it's the right solution for your organization? See [Try GX Cloud](../try_gx_cloud.md).
+
+## Prerequisites
+
+- You have a [GX Cloud Beta account](https://greatexpectations.io/cloud) with [Admin or Editor permissions](../about_gx.md#roles-and-responsibilities).
+
+- You have a PostgreSQL database, schema, and table.
+
+- To improve data security, GX recommends creating a separate PostgreSQL user for your GX Cloud connection.
+
+- You know your PostgreSQL access credentials.
```

Comment:
This line seems a little redundant with the above line

Reply:
@allensallinger you'd be surprised how easily users are confused if you don't include information that we view as redundant.

Reply:
I agree with @allensallinger, the line is unnecessary.